agent/agent.go

```diff
@@ -951,7 +960,7 @@ func (a *Agent) StopInputPlugin(input string, shouldUpdatePlugin bool) error {
 }
 
 // StopOutputPlugin stops an output plugin
-func (a *Agent) StopOutputPlugin(output string) {
+func (a *Agent) StopOutputPlugin(output string, shouldUpdateConfig bool) {
```

Comment:
nit: Probably for all these functions that take in plugin names, instead of naming the parameters just like ```output``` or ```input``` it's probably better to change them to just be ```pluginName``` or ```plugin``` or something.